npm/vite-dev-server/src/makeCypressPlugin.ts

```diff
@@ -40,7 +46,7 @@ export const makeCypressPlugin = (
           tag: 'script',
           injectTo: 'body',
           attrs: { type: 'module' },
-          children: `import(${JSON.stringify(INIT_FILEPATH)})`,
+          children: `import(${JSON.stringify(`${base}@fs/${INIT_FILEPATH}`)})`,
```

Comment:
I don't think we need JSON.stringify. I was able to [remove something similar in webpack-dev-server](https://github.com/cypress-io/cypress/pull/16056/files#diff-fa95d7e53f1c3e9c95d91c3bab0b4692f126b57590abb85f0e13c8644807dccaR20).

Reply:
I am not sure you should have removed it from webpack-dev-server either.
Do test hard cases with special char in the folder name:
- `Oma's sister`. 
- `The "Best" project`
- `The back\slash`

In those cases we will end up with either an unterminated string or some characters that are removed from the string.
It is not my intention.

Reply:
Ooo ok. Maybe we should revert that change I made. :|